File: mario.py
from cs50 import get_int

# get input from user between 1-8 using get_int
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# print out pyramid using its parameters
for i in range(n):
    print(" " * (n-1-i), end="")
    print("#" * (i+1), end="  ")
    print("#" * (i+1), end="")
    print("" * (n-1-i))

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
// get integer between 1 to 8, or program repeats
    int n;
    do
    {
        n = get_int("Height:");
    }
    while (n > 8 || n < 1);
    //make n number of rows
    for (int i = 0; i < n; i++)
    {
        //make left-aligned empty space
        for (int a = n - 1; a > i; a = a - 1)
        {
            printf(" ");
        }
        //make first right-aligned '#' stack
        for (int b = -1; b < i; b++)
        {
            printf("#");
        }
        printf("  ");
        //make second left-aligned '#' stack
        for (int c = -1; c < i; c++)
        {
            printf("#");
        }
        printf("\n");
    }
}